client/components/LearnerViewDirective.js

```diff
@@ -733,6 +737,30 @@ tie.directive('learnerView', [function() {
         };
 
         /**
+          * Highlights syntax errors in the coding UI
+          *
+          * @param {number} lineNumber
+          */
+        var highlightLine = function(lineNumber) {
+          var actualLineNumber = lineNumber - 1;
+          var codeLines = document.querySelectorAll('.CodeMirror-line');
+          codeLines[actualLineNumber].className += ' tie-syntax-error-line';
```

Comment:
Just to check, why do you need to apply the new class to two places? Would just one not work?